Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
   end
 
-  def norris_parse(array)
-    result = ''
-    array.each do |a|
-      if a[0].zero?
-        result += '00 '
-      else
-        result += '0 '
-      end
-      result += '0' * a[1] + ' '
+  def norris_parse(series_array)
+    parsed = series_array.each_with_object('') do |a, result|
+      result << if a[0].zero?
+                  '00 '
+                else
+                  '0 '
+                end
+      result << '0' * a[1] + ' '
     end
-    result.strip
+    parsed.strip
   end
 
   def start
     binary = @message.each_char.each_with_object('') do |char, b|
       b << convert_in_binary(convert_in_ascii(char))
     end
-    array = split_series binary.each_char.map(&:to_i)
-    output = norris_parse array
+    array = split_series(binary.each_char.map(&:to_i))
+    output = norris_parse(array)
     puts output
```

Comment:
Same comment as #1 - you can print the answer after getting the answer outside of the class